### ISSUE TITLE
Warnings options for different connection

### ISSUE THREAD

**Issue by WCByrne** [jakob/Postico#462]
You can probably imagine where this request comes from and I imagine I'm not the only one that would find some use in it. Would be great to add the ability to setup alert severity levels for different connections. When working with multiple connections it can be easy to get mixed up between them. 😉

For a hypothetical situation... let's say you are working on a development/test db and making changes so you get used to just confirming the change prompts, which is fine, because it's a test db. Then you accidentally switch over to the production db and continue on with your development work it can be easy to continue clicking away at the confirmations... 

In other words if this option were off you would get the default [Save Changes], but enable it for the connection and you would get an additional prompt to ask "Are you sure you want to do this". Could be a simple option in the connection setup.

Hope it can be considered in a future update!

**Comment by defkode** [comment]
I'd like the same feature. I use connection color for this (red for production to distinguish it).

It would be fantastic to be able for example to mark connection as "readonly" (even if permissions allows it)

**Comment by jakob** [comment]
The easiest / best way to implement this would probably to allow setting the default transaction mode (see https://github.com/jakob/Postico/issues/207 for details). I think this is what we are going to do.

I'm closing this issue as a duplicate.
